next-app/src/app/edit/page.tsx

```diff
@@ -0,0 +1,253 @@
+          {/* エディター */}
+          <div>
+            {/* エディター部分のヘッダー */}
+            <div className='bg-gray-200 p-4 relative'>
+              <div className='flex'>
+                <div className='bg-gray-800 rounded-lg p-2'>
+                  <p className='text-white font-bold'>Editor</p>
+                </div>
+
+                <div className='flex gap-1 overflow-x-auto'>
+                  <Image
+                    src={IconHash}
+                    alt='icon-#'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconB}
+                    alt='icon-B'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconI}
+                    alt='icon-I'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconU}
+                    alt='icon-U'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconS}
+                    alt='icon-S'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconColor}
+                    alt='icon-Color'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconImage}
+                    alt='icon-Image'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconLink}
+                    alt='icon-Link'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconList}
+                    alt='icon-List'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconNumList}
+                    alt='icon-NumList'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconInline}
+                    alt='icon-inline'
+                    width={40}
+                    height={40}
+                  />
+                  <Image
+                    src={IconBlockcode}
+                    alt='icon-Blockcode'
+                    width={40}
+                    height={40}
+                  />
+                </div>
+              </div>
+
+              {/* エディターとプレビューの切り替え部分 */}
+              <div className='absolute top-1/4 right-[-15px]'>
+                <p className='text-3xl'>⇔</p>
+              </div>
+            </div>
+
+            <div className='p-4  h-full'>
```

Comment:
[必須]
`h-full (height: 100%)` は「親要素の高さの100%」になるため、上部のタブの高さも含んでしまい、現状親要素をはみ出してしまっている

<img width="1822" alt="Screenshot 2024-04-09 at 15 35 20" src="https://github.com/YoslabEngineers/seminar-report-editor/assets/52482280/05149650-fe0a-4606-b78f-df7aba029bb5">

こういうときは、親要素に `flex` を指定し、子要素側に `flex-grow` を指定すると、
子要素を「親要素の残ったスペースいっぱいに広げる」ことができる。

```suggestion
            <div className='flex flex-col gap-3 p-4  flex-grow'>
```

Reply:
5ace172d18e84eb06b13a5d68c8cd37c3b4e8789 で修正しました👍